osf/management/commands/populate_impact_metrics.py

```diff
@@ -0,0 +1,92 @@
+import datetime as dt
+from random import random
+from django.core.management.base import BaseCommand
+
+from osf.metrics import (
+    InstitutionProjectCounts,
+    PreprintView,
+    PreprintDownload,
+    UserInstitutionProjectCounts,
+)
+
+from osf.models import Institution, OSFUser, Preprint
+
+
+# Edit the following guids based off your local environment
+USERS = [OSFUser.load('abd53u'), OSFUser.load('hy84n'), OSFUser.load('7zyg2')]
+PREPRINTS = [Preprint.load('far32'), Preprint.load('67bzg')]
+INSTITUTION = Institution.load('cos')
```

Comment:
These might be good have be optional parameters to the management command and, if they're not provided, are just filled automatically based on database queries. As long as the script prints out the results so you can verify.